pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -879,45 +858,47 @@ async def get_image_info(pullspec: str, raise_if_not_found: bool = False):
                     raise IOError(f"Image {pullspec} is not found.")
                 return None
             raise ChildProcessError(f"Error running {cmd}: exit_code={rc}, stdout={stdout}, stderr={stderr}")
+
+        # Info provided by oc need to be converted back into Skopeo-looking format
         info = json.loads(stdout)
-        if not isinstance(info, dict):
+        if not isinstance(info, list):
             raise ValueError(f"Invalid image info: {info}")
-        return info
+
+        media_types = set([manifest['mediaType'] for manifest in info])
+        if len(media_types) > 1:
+            raise ValueError(f'Inconsistent media types across manifests: {media_types}')
+
+        manifests = {
+            'mediaType': media_types.pop(),
```

Comment:
Here we expect `mediaType` to be something like `application/vnd.docker.distribution.manifest.list.v2+json`. Entries in the manifest list has mediaType `application/vnd.docker.distribution.manifest.v2+json`. They are different.

Reply:
Right. I propose to hardcode the desired value, altough it's not clear who consumes it. We might just as well remove that check